package.json

```diff
@@ -63,7 +63,6 @@
 		"date-fns": "1.30.1",
 		"decompress-zip": "0.3.1",
 		"express": "4.17.1",
-		"extract-loader": "3.0.0",
```

Comment:
Why is this removed in this PR? Just wanted to clean up? Are you 100% sure we don't use it?

Reply:
yeah, this hasnt been used for ages now.